Dockerfile

```
@@ -29,6 +29,12 @@ ENV ALLOW_RESTARTS=0 \
     VOLUMES=0
 COPY haproxy.cfg /usr/local/etc/haproxy/haproxy.cfg
 
+# Install python/pip
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
```

Local binaries are more correctly there:

```suggestion
RUN apk add --update --no-cache python3 && ln -sf python3 /usr/local/bin/python
```

---

Mmmm ok, but why? :thinking: 

In most systems python is in `/usr/bin/python` and, if we opt for this local path, we don't even have `python` in `$PATH`. We would have to change it and, is it really worth it?

---

> Mmmm ok, but why? thinking

It's the [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard).



> if we opt for this local path, we don't even have `python` in `$PATH`

That's not true. :man_shrugging:

---

Ok, thanks for the explanation :+1: 

> That's not true. :man_shrugging:

And sorry, I got confused. Turns out I couldn't use it because of a wrong symlink. Already fixed. Ty!